### DISCUSSION TITLE
【求助】家人们，急需钱！！！

### DISCUSSION THREAD

**Discussion by AL-Pinecore** [share121/inter-knot#242] [其他]
家人们，我老婆出事了现在住进了医院，急需648元手术费！！！
有哪位热心人能帮帮我啊？真的十分感谢啊啊啊啊！！！！！！！
什么，你说叫什么名字？芙宁娜啊你也应该不认识吧？

**Comment by obuyniq** [comment]
这个委托我接了！

![](https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png)

**Reply by KinoZR** [reply]
坏了，你说的这个老婆，好像在我壶里
